README.md

```diff
@@ -30,7 +30,12 @@ There are two main categories of samples: **Camera** and **Applications**. The s
       - [**ReadIterateZDF**][ReadIterateZDF-url] - Read point cloud data from a ZDF file, iterate through it, and extract individual points.
       - [**ZDF2PLY**][ZDF2PLY-url] - Convert point cloud from ZDF a file to a PLY file.
   - **Advanced**
-    - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration.
+    - [**HandEyeCalibration**][HandEyeCalibrationReadme-url]
+        - [**HandEyeCalibration**][HandEyeCalibration-url] - Perform Hand-Eye calibration.
+        - [**UtilizeEyeInHandCalibration**][UtilizeEyeInHandCalibration-url] - Transform single data point or entire point cloud from camera frame to robot base frame using Eye-in-Hand calibration matrix.
+          - **Dependencies:**
```

Comment:
I suggest mentioning here that the user does not need to install this manually, or they'll maybe start doing that before the discover that it just works out of the box.

Reply:
done